github-dark.css

```diff
@@ -5250,3 +5250,94 @@
   }
   /* Literal.Number.Integer.Long */
 }
+
+@-moz-document domain("github.com") {
+  /* ui when there are no workflows setup */
+  .deployments-timeline-item + div:not(.deployments-timeline-item) {
+    background-image: linear-gradient( rgba(28, 28, 28, 0), rgba(28, 28, 28, 1)) !important;
+  }
+  [style*="background-color: #e1e4e8"] {
+    background-color: #8e8e8e !important;
+  }
+  [style*="color: #e1e4e8"] {
+    color: #8e8e8e !important;
+  }
+  /**
+   * this is the same form as when you're committing from the site
+   * this means there's a small arrow on the left (which points at
+   * the user's avatar) and this ends up showing in the dropdown too
+   */
+  .file-commit-form .dropdown-menu .commit-form:after {
+    content: none !important;
+  }
+  .file-commit-form .commit-form {
+    background: #181818 !important;
+  }
+}
+
+@-moz-document domain("launch-editor.github.com") {
```

Comment:
Maybe add a comment line above each such section. Also note that any empty line will trigger lint errors. We might want to relax that stylelint rule to allow one empty line, but I kind of like the compressed view it gives without empty lines.

Reply:
On the other hand, I think I'd actually prefer to not add these `@-moz-document` sections if the selectors are not shared with other sites. It's kind of a hard call because these provide scoping, but blow up style size because of duplication, which in turn can impact performance.

Reply:
Anything inside the `github.com` section should be able to be moved into the main section. What's in the `launch-editor.github.com` section might be able to be moved, but I won't know until I try merging them together.

Reply:
We can keep it in sections if you prefer. I see a few selectors that could cause conflicts like `.overlay` and `input`, so it might be safer that way.